src/error-catch-notifier.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * Error catching is enabled
+ * @member {bool}
+ */
+let _isEnabled = false;
```

Comment:
Might have been better to have this file create a closure rather than letting the state reside directly in the module. I realize this is what we'd discussed doing. The closure approach would probably have been easier to test as it'd permit non-overlapping instances of the state.